### PR TITLE
[HOLD] packages: add instructions on using packages

### PR DESCRIPTION
**Pull request by ChanderG**
Added a few instructions on getting started with using packages. This is information I needed to just write small scripts - getting started with CL and could find only with some trial/error.

### CODE REVIEW

File: packages.md [hunk 1]
```diff
@@ -4,6 +4,44 @@ title: Packages
 
 See: [The Complete Idiot's Guide to Common Lisp Packages][guide]
 
+## Using packages
+
+To install and use a package with a single command, use `ql:quickload` as follows:
+~~~lisp
+(ql:quickload :package)
```
Comment (vindarel) [inline packages.md:11]: I think this sentence misleads packages with systems. 

Also quickload is shown on systems.md / loading a system.

and how to access symbols with `package:function` is shown below at "accessing symbols from a package".

The current recipe shows a package definition, that effectively creates a package.  I wouldn't start this section with a quickload example.

Could you rephrase, or add an introduction explaining the difference between systems and packages, linking to the systems page if necessary?

File: packages.md [hunk 2]
```diff
@@ -4,6 +4,44 @@ title: Packages
 
 See: [The Complete Idiot's Guide to Common Lisp Packages][guide]
 
+## Using packages
+
+To install and use a package with a single command, use `ql:quickload` as follows:
+~~~lisp
```
Comment (vindarel) [inline packages.md:10]: please add a blank line before all the `~~~lisp`, thanks.

**Review by vindarel — CHANGES_REQUESTED** [review]

**Comment by ChanderG** [comment]
> Hi @ChanderG, maybe you needed a quick way to see how to use external libraries ?

Yes - that's exactly right. Maybe this section should go somewhere else and not in this page on packages?

File: packages.md (hunk 1 above)
Comment (ChanderG) [reply packages.md:11]: Yes - to be honest - I haven't yet grasped the difference between those terms.

**Comment by vindarel** [comment]
I sent changes that should answer your concerns. I added sub-bullets in the index to see "install libraries" more quickly and I augmented the content with your input from this PR (mainly the packages page).

**Comment by ChanderG** [comment]
@vindarel Your changes seem to cover everything I was looking for. Thanks!
Will close this PR now.
